docs/components/inline-edit.md

```diff
@@ -0,0 +1,72 @@
+# Inline Edit
+
+**KInlineEdit** - A wrapper which adds inline edit capability. Currently only supports single text input.
+
+<Komponent :data="{ inlineText: 'Click to edit me' }" v-slot="{ data }">
+  <KInlineEdit @changed="newVal => data.inlineText = newVal"><h3>{{ data.inlineText }}</h3></KInlineEdit>
```

Comment:
does the component work with full-width? We will probably need to have it expand beyond the text to be full-width of the container e.g. when you hover over a far right portion of the inline-edit container:
![image](https://user-images.githubusercontent.com/5770711/84696688-dffafd80-af1a-11ea-919a-c37079951662.png)

Reply:
it doesn't currently but it should by default actually. Then you can control the overall width at the top level. Will update

Reply:
Changed this to be 100% by default and added an example to make smaller